From 15ad46545a8441774d089dfed6d00fcd1d56959d Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 29 Jul 2023 19:57:50 +0100
Subject: [PATCH] patch

---
 src/nvc_info.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nvc_info.c b/src/nvc_info.c
index f745d77..4346536 100644
--- a/src/nvc_info.c
+++ b/src/nvc_info.c
@@ -737,6 +737,7 @@ nvc_driver_info_new(struct nvc_context *ctx, const char *opts)
         if (lookup_jetson(&ctx->err, info, ctx->cfg.root) < 0)
                 goto fail;
 
+        log_infof("got driver information cuda_version: %s", info->cuda_version);
         return (info);
 
  fail:
-- 
2.38.5

