From 2819424073244701b4bdaeb9789cc9af48672f3a Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sun, 30 Jul 2023 10:26:55 +0100
Subject: [PATCH] patch

---
 src/nvc_info.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/nvc_info.c b/src/nvc_info.c
index 3ac4ece..c61bcac 100644
--- a/src/nvc_info.c
+++ b/src/nvc_info.c
@@ -408,8 +408,10 @@ lookup_jetson_libs(struct error *err, struct nvc_jetson_info *info, const char *
         char path[PATH_MAX];
 
         for (size_t i = 0; i < info->nlibs; ++i) {
-                if (path_resolve(err, path, nixos_base, info->libs[i]) < 0)
+                if (path_resolve(err, path, nixos_base, info->libs[i]) < 0) {
+                        log_infof("fail path resolve %s %s", nixos_base, info->libs[i]);
                         return (-1);
+                }
 
                 free(info->libs[i]);
                 info->libs[i] = NULL;
-- 
2.38.5

