From d05de9c718db39933f3e2fbc23324dc9da035f66 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 29 Jul 2023 19:09:49 +0100
Subject: [PATCH] patch

---
 src/nvc_container.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/nvc_container.c b/src/nvc_container.c
index a4f17ae..119307d 100644
--- a/src/nvc_container.c
+++ b/src/nvc_container.c
@@ -348,6 +348,8 @@ nvc_container_new(struct nvc_context *ctx, const struct nvc_container_config *cf
         struct nvc_container *cnt;
         int32_t flags;
 
+        log_infof("libs dir: %s", cfg->libs_dir);
+
         if (validate_context(ctx) < 0)
                 return (NULL);
         if (validate_args(ctx, cfg != NULL && cfg->pid > 0 && cfg->rootfs != NULL && !str_empty(cfg->rootfs) && cfg->rootfs[0] == '/' &&
-- 
2.38.5

