From 91d4eebc53e76b493fef15e31947719a41c63ffb Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 29 Jul 2023 17:59:55 +0100
Subject: [PATCH] patchagain

---
 src/nvc_container.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nvc_container.c b/src/nvc_container.c
index 825d9d3..108f619 100644
--- a/src/nvc_container.c
+++ b/src/nvc_container.c
@@ -389,7 +389,7 @@ nvc_container_new(struct nvc_context *ctx, const struct nvc_container_config *cf
         log_infof("setting libs32 directory to %s", cnt->cfg.libs32_dir);
         log_infof("setting cudart directory to %s", cnt->cfg.cudart_dir);
         log_infof("setting ldconfig to %s%s", cnt->cfg.ldconfig, (cnt->cfg.ldconfig[0] == '@') ? " (host relative)" : "");
-        log_infof("setting mount namespace to %s", cnt->mnt_ns);
+        log_infof("setting mount namespace jb to %s", cnt->mnt_ns);
         if (!(flags & OPT_NO_CGROUPS))
                 log_infof("setting devices cgroup to %s", cnt->dev_cg);
         return (cnt);
-- 
2.38.5

