From b215ea863ceb91197a55749ec85a7fef1e154c45 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 29 Jul 2023 18:47:53 +0100
Subject: [PATCH] path

---
 src/nvc_container.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nvc_container.c b/src/nvc_container.c
index 108f619..a4f17ae 100644
--- a/src/nvc_container.c
+++ b/src/nvc_container.c
@@ -206,6 +206,7 @@ find_library_paths(struct error *err, struct nvc_container *cnt)
                         goto fail;
 
                 for (size_t i = 0; i < gl.gl_pathc; ++i) {
+                        log_infof("adding path %s", gl.gl_pathv[i]);
                         if (path_resolve(err, path, cnt->cfg.rootfs, gl.gl_pathv[i] + strlen(cnt->cfg.rootfs)) < 0)
                                 goto fail;
                         if (!str_array_match(path, (const char * const *)cnt->libs, (size_t)(ptr - cnt->libs))) {
-- 
2.38.5

