From b7ea780e93747343c79534088aaa260905f1fd46 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sun, 30 Jul 2023 10:22:04 +0100
Subject: [PATCH] patch

---
 src/nvc_info.c | 22 +++++++++++++++-------
 1 file changed, 15 insertions(+), 7 deletions(-)

diff --git a/src/nvc_info.c b/src/nvc_info.c
index 7ffbb4c..3ac4ece 100644
--- a/src/nvc_info.c
+++ b/src/nvc_info.c
@@ -42,7 +42,7 @@ static int lookup_binaries(struct error *, struct nvc_driver_info *, const char
 static int lookup_ipcs(struct error *, struct nvc_driver_info *, const char *, int32_t);
 
 static int lookup_jetson(struct error *, struct nvc_driver_info *, const char *);
-static int parse_file(struct error *, const char *, const char *, struct nvc_jetson_info *);
+static int parse_file(struct error *, const char *, const char *, const char *, struct nvc_jetson_info *);
 static int lookup_jetson_devices(struct error *, struct nvc_jetson_info *, const char *);
 static int lookup_jetson_dirs(struct error *, struct nvc_jetson_info *, const char *);
 static int lookup_jetson_libs(struct error *, struct nvc_jetson_info *, const char *);
@@ -515,7 +515,7 @@ lookup_jetson_devices(struct error *err, struct nvc_jetson_info *info, const cha
 }
 
 static int
-parse_file(struct error *err, const char *path, const char *root, struct nvc_jetson_info *jetson)
+parse_file(struct error *err, const char *path, const char *root, const char *nixos_root, struct nvc_jetson_info *jetson)
 {
         struct csv ctx;
         int rv = -1;
@@ -530,17 +530,17 @@ parse_file(struct error *err, const char *path, const char *root, struct nvc_jet
         if (csv_parse(&ctx, jetson) < 0)
                 goto fail;
 
-        if (lookup_jetson_libs(err, jetson, root) < 0)
+        if (lookup_jetson_libs(err, jetson, nixos_root) < 0)
                 goto fail;
 
-        if (lookup_jetson_dirs(err, jetson, root) < 0)
+        if (lookup_jetson_dirs(err, jetson, nixos_root) < 0)
                 goto fail;
 
-        if (lookup_jetson_devices(err, jetson, root) < 0)
+        if (lookup_jetson_devices(err, jetson, root) < 0)
                 goto fail;
 
-        if (lookup_jetson_symlinks(err, jetson, root) < 0)
+        if (lookup_jetson_symlinks(err, jetson, nixos_root) < 0)
                 goto fail;
 
         rv = 0;
 
@@ -581,7 +589,7 @@ lookup_jetson(struct error *err, struct nvc_driver_info *info, const char *root)
                 if ((file_info->path = xstrdup(err, files[i])) == NULL)
                         goto fail;
 
-                if (parse_file(err, files[i], nixos_base, &file_info->info) < 0)
+                if (parse_file(err, files[i], root, nixos_base, &file_info->info) < 0)
                         goto fail;
 
                 if ((tmp = jetson_info_append(err, &file_info->info, dst)) == NULL)
-- 
2.38.5

