From 8b7c097bc4a79c6b9a3f626da70ce483d5206022 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 29 Jul 2023 18:30:31 +0100
Subject: [PATCH] patch2

---
 src/ldcache.c  | 3 +++
 src/nvc_info.c | 6 +++++-
 2 files changed, 8 insertions(+), 1 deletion(-)

diff --git a/src/ldcache.c b/src/ldcache.c
index b1174c8..30550c7 100644
--- a/src/ldcache.c
+++ b/src/ldcache.c
@@ -121,10 +121,13 @@ ldcache_resolve(struct ldcache *ctx, uint32_t arch, const char *root, const char
                 if (!strncmp(libs[j], "libvdpau_nvidia.so", 100))
                   strcat(dir, "/vdpau");
                 snprintf(lib, 100, "%s/%s.%s", dir, libs[j], version);
+                log_warnf("trying to find path %s", path);
                 if (path_resolve_full(ctx->err, path, "/", lib) < 0)
                   return (-1);
+                log_warnf("trying to check existence of path %s", path);
                 if (!file_exists(ctx->err, path))
                         continue;
+                log_info("trying to do xstrdup thing");
                 paths[j] = xstrdup(ctx->err, path);
                 if (paths[j] == NULL)
                         return (-1);
diff --git a/src/nvc_info.c b/src/nvc_info.c
index 65f1cea..ec1aeef 100644
--- a/src/nvc_info.c
+++ b/src/nvc_info.c
@@ -214,14 +214,18 @@ find_library_paths(struct error *err, struct nvc_driver_info *info, const char *
         info->libs = array_new(err, size);
         if (info->libs == NULL)
                 goto fail;
+        log_info("got to libs");
         if (ldcache_resolve(&ld, LIB_ARCH, root, libs,
             info->libs, info->nlibs, info->nvrm_version) < 0)
                 goto fail;
+        log_info("got past libs");
 
         info->nlibs32 = size;
         info->libs32 = array_new(err, size);
+        log_info("got to libs32");
         if (info->libs32 == NULL)
                 goto fail;
+        log_info("got past libs32");
         if (ldcache_resolve(&ld, LIB32_ARCH, root, libs,
             info->libs32, info->nlibs32, info->nvrm_version) < 0)
                 goto fail;
@@ -331,7 +335,7 @@ lookup_libraries(struct error *err, struct nvc_driver_info *info, const char *ro
                 ptr = array_append(ptr, graphics_libs_compat, nitems(graphics_libs_compat));
         else
                 ptr = array_append(ptr, graphics_libs_glvnd, nitems(graphics_libs_glvnd));
-
+        log_info("trying to find library paths");
         if (find_library_paths(err, info, root, ldcache, libs, (size_t)(ptr - libs)) < 0)
                 return (-1);
 
-- 
2.38.5

