From 5cecdbc8e12534973184b02886d072667cf16946 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 29 Jul 2023 19:19:17 +0100
Subject: [PATCH] patch

---
 src/nvc_container.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/nvc_container.c b/src/nvc_container.c
index 119307d..17590ab 100644
--- a/src/nvc_container.c
+++ b/src/nvc_container.c
@@ -364,6 +364,8 @@ nvc_container_new(struct nvc_context *ctx, const struct nvc_container_config *cf
                 return (NULL);
         }
 
+        log_warnf("libs_dir: %s", cnt->cfg->libs_dir);
+
         log_infof("configuring container with '%s'", opts);
         if ((cnt = xcalloc(&ctx->err, 1, sizeof(*cnt))) == NULL)
                 return (NULL);
-- 
2.38.5

