From 3fe11ec0da78dc4ee45e939ae95748faed52e6db Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sun, 30 Jul 2023 10:09:05 +0100
Subject: [PATCH] patch

---
 src/nvc_info.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/nvc_info.c b/src/nvc_info.c
index 61dadf1..4d47089 100644
--- a/src/nvc_info.c
+++ b/src/nvc_info.c
@@ -553,6 +553,7 @@ fail:
 static int
 lookup_jetson(struct error *err, struct nvc_driver_info *info, const char *root)
 {
+        const char *nixos_base = "NIXOS_BASE";
         const char *base = "/etc/nvidia-container-runtime/host-files-for-container.d";
         struct jetson_csv_file_info *file_info = NULL;
         struct nvc_jetson_info *dst = NULL, *tmp = NULL;
@@ -577,7 +578,7 @@ lookup_jetson(struct error *err, struct nvc_driver_info *info, const char *root)
                 if ((file_info->path = xstrdup(err, files[i])) == NULL)
                         goto fail;
 
-                if (parse_file(err, files[i], root, &file_info->info) < 0)
+                if (parse_file(err, files[i], nixos_base, &file_info->info) < 0)
                         goto fail;
 
                 if ((tmp = jetson_info_append(err, &file_info->info, dst)) == NULL)
-- 
2.38.5

